solution/serverless-test-files/serverless-backend.yml

```diff
@@ -0,0 +1,237 @@
+    handler: migrate.handler
+#    layers:
+#      - ${cf:cmcs-eregs-static-assets-dev.PythonRequirementsLambdaLayerQualifiedArn}
+    timeout: 300
+  create_su:
+    handler: createsu.handler
+#    layers:
+#      - ${cf:cmcs-eregs-static-assets-dev.PythonRequirementsLambdaLayerQualifiedArn}
+    timeout: 300
+#  populate_content:
+#   environment:
+#      SUPPLEMENTAL_CONTENT_PATH: /var/task/guidance
+#    handler: populate_content.handler
+#    layers:
+#      - ${cf:cmcs-eregs-static-assets-dev.PythonRequirementsLambdaLayerQualifiedArn}
+#    timeout: 300
+
+
+resources:
+
+  Resources:
+
+    LambdaFunctionRole:
+      Type: AWS::IAM::Role
+      Properties:
+        Path: "/delegatedadmin/developer/"
+        AssumeRolePolicyDocument:
+          Version: '2012-10-17'
+          Statement:
+            - Effect: Allow
+              Principal:
+                Service:
+                  - lambda.amazonaws.com
+              Action: sts:AssumeRole
+        # note that these rights are needed if you want your function to be able to communicate with resources within your vpc
+        #PermissionsBoundary:
+          #Fn::Join:
+            #- ''
+            #- - 'arn:aws:iam::'
+              #- Ref: AWS::AccountId
+              #- ':policy'
+              #- '/cms-cloud-admin/developer-boundary-policy'
+        #ManagedPolicyArns:
+          #- arn:aws:iam::aws:policy/service-role/AWSLambdaVPCAccessExecutionRole
+        #Policies:
+        #  - PolicyName: LambdaPolicy
+        #    PolicyDocument:
+        #      Version: '2012-10-17'
+        #      Statement:
+        #        - Effect: Allow # note that these rights are given in the default policy and are required if you want logs out of your lambda(s)
+        #          Action:
+        #            - logs:CreateLogGroup
+        #            - logs:CreateLogStream
+        #            - logs:PutLogEvents
+        #          Resource:
+        #            - 'Fn::Join':
+        #              - ':'
+        #              -
+        #                - 'arn:aws:logs'
+        #                - Ref: 'AWS::Region'
+        #                - Ref: 'AWS::AccountId'
+        #                - 'log-group:/aws/lambda/*:*:*'
+        #        -  Effect: "Allow"
+        #           Action:
+        #             - "s3:PutObject"
+        #           Resource:
+        #             Fn::Join:
+        #               - ""
+        #               - - "arn:aws:s3:::"
+        #                 - "Ref" : "ServerlessDeploymentBucket"
+
+    ServerlessSecurityGroup:
+      Type: AWS::EC2::SecurityGroup
+      Properties:
+        GroupDescription: SecurityGroup for Serverless Functions
+        VpcId: 'vpc-079fb0427309a063e'
+        Tags:
+          - Key: "Name"
+            Value: "ServerlessSecurityGroup"
+
+    DBSecurityGroup:
+      Type: AWS::EC2::SecurityGroup
+      Properties:
+        GroupDescription: SecurityGroup for Database
+        VpcId: 'vpc-079fb0427309a063e'
```

Comment:
were these VpcId's created in aws console manually? If so the document should reflect that a vpc needs to be created and that this file has to be updated with that VpcId.